### PR TITLE
INTMDB-904: HELP: mongodbatlas_org_invitation is missing ORG_BILLING_READ_ONLY role support

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description
Ticket: [INTMDB-904](https://jira.mongodb.org/browse/INTMDB-904)
Link to any related issue(s): #1280

This PR removes the hardcoded validation on the roles field. We will rely on the API from now on.

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

**Review by martinstibbe — APPROVED** [review]
LGTM

File: website/docs/d/org_invitation.html.markdown [hunk 1]
```diff
@@ -40,11 +40,6 @@ In addition to the arguments, this data source exports the following attributes:
 * `expires_at` - Timestamp in ISO 8601 date and time format in UTC when the invitation expires. Users have 30 days to accept an invitation.
 * `inviter_username` - Atlas user who invited `username` to the organization.
 * `teams_ids` - An array of unique 24-hexadecimal digit strings that identify the teams that the user was invited to join.
-* `roles` - Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. The following options are available:
-  * ORG_OWNER
-  * ORG_GROUP_CREATOR
-  * ORG_BILLING_ADMIN
-  * ORG_READ_ONLY
-  * ORG_MEMBER
+* `roles` - Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. The [MongoDB Documentation](https://docs.atlas.mongodb.com/reference/user-roles/#organization-roles) describes the roles a user can have.
```
Comment (jwilliams-mongo) [inline website/docs/d/org_invitation.html.markdown]: nitpick: we moved the docs to a different location a while back. We should use the current URL to cut down on redirects and improve SEO
```suggestion
* `roles` - Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles a user can have.
```

File: website/docs/r/org_invitation.html.markdown [hunk 2]
```diff
@@ -12,13 +12,7 @@ description: |-
...
 Each invitation for an Atlas user includes roles that Atlas grants the user when they accept the invitation.
 
-The [MongoDB Documentation](https://docs.atlas.mongodb.com/reference/user-roles/#organization-roles) describes the roles a user can have, which map to:
-
-* ORG_OWNER
-* ORG_GROUP_CREATOR
-* ORG_BILLING_ADMIN
-* ORG_READ_ONLY
-* ORG_MEMBER
+The [MongoDB Documentation](https://docs.atlas.mongodb.com/reference/user-roles/#organization-roles) describes the roles a user can have.
```
Comment (jwilliams-mongo) [inline website/docs/r/org_invitation.html.markdown]: see above
```suggestion
The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles a user can have.
```

File: website/docs/r/org_invitation.html.markdown [hunk 3]
```diff
@@ -57,12 +51,7 @@ resource "mongodbatlas_org_invitation" "test1" {
 * `org_id` - (Required) Unique 24-hexadecimal digit string that identifies the organization to which you want to invite a user.
 * `username` - (Required) Email address of the invited user. This is the address to which Atlas sends the invite. If the user accepts the invitation, they log in to Atlas with this username.
 * `teams_ids` - (Optional) An array of unique 24-hexadecimal digit strings that identify the teams that the user was invited to join.
-* `roles` - (Required) Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. The following options are available:
-  * ORG_OWNER
-  * ORG_GROUP_CREATOR
-  * ORG_BILLING_ADMIN
-  * ORG_READ_ONLY
-  * ORG_MEMBER
+* `roles` - (Required) Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. The [MongoDB Documentation](https://docs.atlas.mongodb.com/reference/user-roles/#organization-roles) describes the roles a user can have.
```
Comment (jwilliams-mongo) [inline website/docs/r/org_invitation.html.markdown]:
```suggestion
* `roles` - (Required) Atlas roles to assign to the invited user. If the user accepts the invitation, Atlas assigns these roles to them. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles a user can have.
```

**Review by jwilliams-mongo — APPROVED** [review]
@andreaangiolillo LGTM with a few suggestions to use current doc links

**Pull request merged**